src/ngMock/angular-mocks.js

```diff
@@ -135,6 +140,52 @@ angular.mock.$Browser = function() {
   self.baseHref = function() {
     return this.$$baseHref;
   };
+
+  self.interval = function(fn, interval) {
+    self.interval.repeatFns.push({
+      nextTime:(self.interval.now + interval),
+      delay: interval,
+      fn: fn,
+      id: self.interval.nextRepeatId
+    });
+    self.interval.repeatFns.sort(function(a,b) { return a.nextTime - b.nextTime;});
+
+    return self.interval.nextRepeatId++;
+  };
+
+  self.interval.cancel = function(id) {
+    var fnIndex;
+
+    angular.forEach(self.interval.repeatFns, function(fn, index) {
+      if (fn.id === id) fnIndex = index;
+    });
+
+    if (fnIndex !== undefined) {
+      self.interval.repeatFns.splice(fnIndex, 1);
+      return true;
+    }
+
+    return false;
+  };
+
+  self.interval.flush = function(millis) {
+    self.interval.now += millis;
+    while (self.interval.repeatFns.length &&
+        self.interval.repeatFns[0].nextTime <= self.interval.now) {
+      var task = self.interval.repeatFns[0];
+      task.fn();
+      task.nextTime += task.delay;
+      self.interval.repeatFns.sort(function(a,b) { return a.nextTime - b.nextTime;});
+    }
+    return millis;
+  };
+
+  self.interval.repeatFns = [];
+  self.interval.nextRepeatId = 0;
+  self.interval.now = 0;
+
+  self.shutdown = angular.noop;
```

Comment:
Doesn't this mean that timeouts and intervals will continue to work even after shutdown?
I wonder if this could affect someone's tests.